components/skus/browser/rust/wasm/pkg/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@brave-intl/skus-sdk",
```

Comment:
should this name be updated? (to not include `brave-intl`)? ex: `skus-sdk-wasm` (and then probably matching name in components/skus/browser/rust/wasm/Cargo.toml)

Reply:
I think this should be okay since the non-organization part of the NPM package path matches, but I haven't tried publishing it yet. As part of moving this codebase into brave-core we'll be switching to actually publishing the sdk on npmjs.com so I think we want an org prefix for that. ( see e.g. https://www.npmjs.com/package/@brave-intl/bat-usermodel )

Reply:
I think this should be okay since the non-organization part of the NPM package path matches, but I haven't tried publishing it yet. As part of moving this codebase into brave-core we'll be switching to actually publishing the sdk on npmjs.com so I think we want an org prefix for that. ( see e.g. https://www.npmjs.com/package/@brave-intl/bat-usermodel )